### ISSUE TITLE
`isSpinning` flag

### ISSUE THREAD

**Issue by inorganik** [fgnass/spin.js#243]
I love this lib, thanks a lot!

It would be super helpful if there was a boolean property of the spinner that reflected it if was spinning or not. I start and stop spinners, so just checking for the existence of a spinner doesn't work. Sometimes, spinners get called redundantly and I get spinner overlap. An `isSpinning` flag would really help. Thanks.

**Comment by c0r3yz** [comment]
Since no one else seems to have addressed this, and I was looking for similar functionality, I thought I would take a stab at it.  Here's my approach: 

In Sping.js, add an isSpinning property to the Spinner Prototype.  Find this line:

    merge(Spinner.prototype, {

...then add: 

    isSpinning: false,

Inside the Spin function, you need to set isSpinning to true, like so:

    spin: function (target) {
            this.stop()
            this.isSpinning = true;
    ....

and inside the Stop function, you set it to false:

    stop: function () {
            var el = this.el
            if (el) {
                clearTimeout(this.timeout)
                if (el.parentNode) el.parentNode.removeChild(el)
                this.el = undefined
            }
            this.isSpinning = false;
            return this
        },

Then in your code you reference it like so:

    If(mySpinner.isSpinning) {
        ....
    }

I hope that helps!

**Comment by fgnass** [comment]
You can tell if a spinner is spinning by testing for the `el` property since calling `stop()` will reset it to `undefined`:

```
var spinner = new Spinner();
var isSpinning = !!spinner.el;
```

**Comment by TimothyGu** [comment]
Fixed in #286.
